### ISSUE TITLE
Fixes for WCAG 2.0 AA Rating

### ISSUE THREAD

**Issue by BenJam** [mysociety/fixmystreet#1348]
A lot of local government sites are or aspire to be WCAG 2.0 AA rated. We go to a great deal of effort to ensure that the user experience is accessible so let's carry that through. 

A quick check on http://achecker.ca/checker/index.php (which might not parse correctly, alternatives available) finds:
- [x] a minor header nesting issue on the homepage/report pages
- [x] file input on report pages missing a label

And from elsewhere:
- [x] Contrast of front page numbers
